edd-downloads-as-services.php

```diff
@@ -204,10 +204,8 @@ public function is_service( $item_id ) {
 
 			$is_service = get_post_meta( $item_id, '_edd_das_enabled', true );
 
-			// get payment
-			$payment   = get_post( $edd_receipt_args['id'] );
-			$meta      = isset( $payment ) ? edd_get_payment_meta( $payment->ID ) : '';
-			$cart      = isset( $payment ) ? edd_get_payment_meta_cart_details( $payment->ID, true ) : '';
+			// get cart details
+			$cart = edd_get_payment_meta_cart_details( $edd_receipt_args['id'], true );
```

Comment:
It looks like this line is now throwing the following notice (twice) during checkout (not subsequently when viewing the receipt):

```
PHP Notice:  Trying to access array offset on value of type null in \wp-content\plugins\edd-downloads-as-services\edd-downloads-as-services.php on line 208
```

Logging the `$edd_receipt_args` variable at this point, it returns an empty value. Double checked and the error is also present in `master`, so it's not been introduced with this PR, but thinking it would be good to fix while we're in here.

Reply:
Fixed in https://github.com/easydigitaldownloads/edd-downloads-as-services/pull/11/commits/5b62e7f2f9aca66c4d62eb54257bdf6e29c31de0